spec/bin/synapse_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe 'parseconfig' do
+  it 'parses a templated config' do
+    allow_any_instance_of(Synapse::Synapse).to receive(:run).and_return(nil)
```

Comment:
nit: stubbing a call will already make it return nil. we can remove `and_return(nil)`